docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
I'm going to clone directly from the 18f repo instead of my fork - just to make sure.

1. Cloned 18f/web-desing-standards
1. `$ cd ~/path/to/repo`
1. Completed
1. Ran `bundle update`
1. Completed
1. Ran `npm update`
1. Completed
1. Ran `$ gulp website:serve`
1. Halted with: `jekyll 3.1.3 | Error: File to import not found or unreadable: lib/bourbon. Load path: /path/to/repo/docs/_scss on line 3`

It appears `/src/stylesheets/lib` no longer exists: https://github.com/18F/web-design-standards/tree/18f-pages-staging/src/stylesheets

On the initial clone, `/docs/_scss/lib` does exist: https://github.com/18F/web-design-standards/tree/18f-pages-staging/docs/_scss

When `$ gulp website:serve` removes `/docs/_scss` the `/docs/_scss/lib` directory is removed and not replaced.

Let me know if you experience something different - or, of course, if I'm doing this incorrectly.